### PR TITLE
Use github-pages 2.1.4

### PR DESCRIPTION
**Pull request by olleolleolle**
This PR changes the github-pages gem in **use** to point to the **patched-but-not-released version** in the PR linked in the code comment: https://github.com/github/pages-gem/pull/762

- This gets rid of the Kramdown CVE report

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
 source 'https://rubygems.org'
-gem 'github-pages', group: :jekyll_plugins
-
+# Bundler 1.x workaround: Remove this when we are on Bundler 2
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+# When this is merged and released, go back to regular gem: https://github.com/github/pages-gem/pull/762
+gem 'github-pages', github: "github/pages-gem", branch: "hubot/kramdown-v2-3-1", group: :jekyll_plugins
```
Comment (teoljungberg) [inline Gemfile:5]: This has since been merged, so what if we instead point towards their master branch?

Comment (dentarg) [reply Gemfile:5]: I think we should just wait for a new release: https://github.com/github/pages-gem/releases

shouldn't be that long before it happens?

IIRC this site runs on GitHub Pages so it's not like we control that anyway

**Comment by olleolleolle** [comment]
Thanks, @dentarg and @teoljungberg! Now, the gem got released, I edited the PR, and we are rolling again.

**Comment by teoljungberg** [comment]
Excellent 👏🏻

**Pull request merged**